data.json

```diff
@@ -7496,6 +7517,7 @@
             "symlinks": [
                 "accessories-character-map",
                 "applications-fonts",
+                "applixware",
```

Comment:
I'm not sure this is in the right place? It doesn't seem to be to do with font or character mapping from what I can find

Reply:
Given [appixware](https://www.linux.com/news/last-look-applixware/) was reaching EOL in 2006 this is probably moot, but I'm gonna split it off anyways